ansible/roles/pulp-certs/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
 - name: Configure openssl subjectAltName
   lineinfile: dest=/etc/pki/tls/openssl.cnf
               insertafter="^\[ usr_cert \]"
-              line="subjectAltName=DNS:{{ ansible_fqdn }},DNS:{{ ansible_nodename }},DNS:{{ ansible_hostname }}"
```

Comment:
I need to double check this, I don't *think* it was accidental but I don't remember why it was done.